validations/src/validations.rs

```diff
@@ -1155,11 +1315,7 @@ pub fn validate_tally_transaction<'a>(
     }
 
     let expected_collateral_value = if commits_count > 0 {
-        (if dr_state.data_request.collateral == 0 {
-            collateral_minimum
-        } else {
-            dr_state.data_request.collateral
-        }) * u64::from(dr_state.data_request.witnesses)
+        collateral * u64::from(dr_state.data_request.witnesses)
```

Comment:
why do you remove the minimum collateral?

Reply:
`collateral` is defined as

```rust
    let collateral = if dr_state.data_request.collateral == 0 {
        collateral_minimum
    } else {
        dr_state.data_request.collateral
    };
```

So this change should be equivalent.